cmd/ocm/create/cluster/cmd.go

```diff
@@ -551,7 +552,9 @@ func preRun(cmd *cobra.Command, argv []string) error {
 		return err
 	}
 
-	versions, defaultVersion, err := getVersionOptionsWithDefault(connection, args.channelGroup)
+	gcpMarketplaceEnabled := strconv.FormatBool(isGcpMarketplaceSubscriptionType)
+	versions, defaultVersion, err := getVersionOptionsWithDefault(connection, args.channelGroup,
+		&gcpMarketplaceEnabled)
```

Comment:
This will cause filtering by gcp_marketplace_enabled = 'false' when selecting billing models other than `marketplace-gcp` - which is not the goal, PTAL

Reply:
Fixed the logic